lib/pf/web/dispatcher.pm

```diff
@@ -43,6 +43,9 @@ use pf::Portal::Session;
 use pf::web::externalportal;
 use pf::inline;
 
+use pfconfig::cached_scalar;
+tie our $portal_request_timeout, 'pfconfig::cached_scalar', 'resource::portal_request_timeout';
```

Comment:
Since this is already in $Config{captive_portal}{request_timeout}
Why the need to add another namespace?